web-fixtures/sub-folder/cookie_page4.php

```diff
@@ -0,0 +1,17 @@
+<?php
+if (!isset($cookieValue)) {
+    $cookieValue = 'srv_var_is_set';
+}
+
+setcookie('srvr_cookie', $cookieValue, null, '/sub-folder/');
```

Comment:
Using path from DocumentRoot would only work when running a test suite with PHP built-in web server.

Somehow the `$_SERVER['WEB_FIXTURES_HOST']` needs to be incorporated (when present) to support running test suite in existing Web Server's sub-folder.